team-frontend/src/components/RegisterTeam.vue

```diff
@@ -4,7 +4,7 @@
       <b-row>
         <h1>Registreer hier!</h1>
       </b-row>
-      <hr>
+      <hr />
```

Comment:
Ik heb ook de neiging dit te doen (ik neem even aan dat je dit handmatig hebt gedaan), maar tegenwoordig (HTML5) werkt het anders: https://www.w3schools.com/tags/tag_hr.asp. HTML5 is (geloof ik) geen XHTML. Zie ook https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr en https://www.w3schools.com/tags/tag_html.asp

Reply:
Heb ik niet gedaan. Misschien bij het auto-formatten van een document?

Reply:
hm raar, dan hebben we wss niet dezelfde config qua formatting. Blijft geklooi he :-( Anyway, gewoon negeren maar dan, mijn gezeur :)